packages/jsii/lib/compiler.ts

```diff
@@ -72,28 +72,79 @@ export class Compiler implements Emitter {
   }
 
   /**
-     * Compiles the configured program.
-     *
-     * @param files can be specified to override the standard source code location logic. Useful for example when testing "negatives".
-     */
+   * Adds a listener for the 'result' event that is called with EmitResult.
```

Comment:
Document the meaning of the `result` event. It is not super intuitive by the name that this event is emitted every the code is compiled

Reply:
Yeah I actually have a better name for it now... I'll call it `compilationComplete` I think